slideShow.js

```diff
@@ -0,0 +1,117 @@
+// json取得先
+const fetchURL = 'https://jsondata.okiba.me/v1/json/IOQ9t210418034754'
+
+// DOMの取得
+const slideShow = document.getElementById('js-slideShow')
+const slideShowContainer = document.getElementById('js-slideShowContainer')
+const prevArrow = document.getElementById('js-prevArrow')
+const nextArrow = document.getElementById('js-nextArrow')
+const pageNation = document.getElementById('js-pageNation')
+
+// クラス名の追加
+slideShow.classList.add('slideShow')
+slideShowContainer.classList.add('slideShowContainer')
+prevArrow.classList.add('hidden')
+nextArrow.classList.add('hidden')
+
+let currentNum = 0
+const slideImageArray = []
+
+const myFetch = async (fetchURL) => {
+  try {
+    const res = await fetch(fetchURL)
+    const data = await res.json()
+    return data
+  } catch (e) {
+    console.log(e.error)
```

Comment:
多分やりたいことはこういうことだと思うのですが
```suggestion
    console.error(e.error)
```

ここではユーザーに何かAPI都合のエラーが起きたことを示したいのでコンソールというよりはページ内で表現したいです

Reply:
確かにconsoleみないですよね。修正します。